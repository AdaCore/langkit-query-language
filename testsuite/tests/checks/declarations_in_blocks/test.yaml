driver: 'checker'
rule_name: Declarations_In_Blocks
input_sources: ['declarations_in_blocks.adb']
