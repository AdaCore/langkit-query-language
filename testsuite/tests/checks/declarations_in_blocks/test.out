declarations_in_blocks.adb:5:8: rule violation: block statement with local declaration
5 |        declare                       --  FLAG
  |        ^^^^

